consumer/aiml/build/mainline.md

```diff
@@ -0,0 +1,94 @@
+# AI_ML board setup with mainline kernel and Linaro Debian image
```

Comment:
Missing header, see [this](https://raw.githubusercontent.com/96boards/documentation/master/consumer/hikey/hikey960/build/open-embedded.md) for example

Reply:
Added